shared/src/tx_pool/pool.rs

```diff
@@ -212,6 +212,16 @@ impl TxPool {
         self.proposed.get_tx(id).cloned()
     }
 
+    pub fn get_tx_from_proposed_and_others(&self, id: &ProposalShortId) -> Option<Transaction> {
+        self.proposed
+            .get_tx(id)
+            .or_else(|| self.gap.get_tx(id))
+            .or_else(|| self.pending.get_tx(id))
+            .or_else(|| self.orphan.get_tx(id))
+            .or_else(|| self.conflict.get(id).map(|e| &e.transaction))
+            .cloned()
+    }
+
```

Comment:
What is the difference between `get_tx` and `get_tx_from_proposed_and_others`? If there is, please add some comments.

Reply:
I added this new fn according to review comment: https://github.com/nervosnetwork/ckb/pull/1308#discussion_r308047955

Reply:
It looks like they are the same.